offchain/dispatcher/src/dispatcher.rs

```diff
@@ -25,7 +25,7 @@ pub async fn start(
     config: DispatcherConfig,
     metrics: DispatcherMetrics,
 ) -> Result<(), DispatcherError> {
-    info!("Setting up dispatcher with config: {:?}", config);
+    info!("Setting up dispatcher");
```

Comment:
We could remove this log since it doesn't add much else.

Reply:
Sure. Could that be a "trace" instead of "info" then, as logs also acts as "execution markers" that helps spot bugs. What do you think?

Reply:
Changing it to trace is fine as well.

Reply:
@gligneul @fmoura, are we going to keep the `info` instead of changing it to a `trace`?